From 16abd091ca7f91859ae7615f679321fc5a1c863b Mon Sep 17 00:00:00 2001
From: "beiling.xie" <xiekunxun@huawei.com>
Date: Fri, 2 Sep 2022 22:33:05 +0800
Subject: [PATCH] support openssl for dsoftbus

Signed-off-by: beiling.xie <xiekunxun@huawei.com>
---
 BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

diff --git a/BUILD.gn b/BUILD.gn
index 68c53958c..8d9812046 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -31,7 +31,6 @@ config("crypto_config") {
     "-pthread",
     "-fPIC",
     "-Wa,--noexecstack",
-    "-Qunused-arguments",
     "-O3",
     "-DOPENSSL_USE_NODELETE",
     "-DOPENSSL_PIC",
-- 
2.25.1

