--- a/BUILD.gn
+++ b/BUILD.gn
@@ -117,12 +117,13 @@ if (defined(ohos_lite)) {
   ohos_shared_library("mbedtls_shared") {
     public_configs = [ ":mbedtls_config" ]
     output_name = "mbedtls"
-    subsystem_name = "common"
+    subsystem_name = "communication"
     part_name = "dsoftbus_standard"
     sources = MBEDTLS_SOURCES
   }
 
   group("mbedtls") {
+    public_configs = [ ":mbedtls_config" ]
     public_deps = [ ":mbedtls_shared" ]
   }
 }
