From 6d99f180cef13bc66d7286e96b480764e9aa75f9 Mon Sep 17 00:00:00 2001
From: hmilylmk <liumingkai@huawei.com>
Date: Wed, 24 Aug 2022 19:44:32 +0800
Subject: [PATCH] Adaptation for dsoftbus v3.1

Signed-off-by: hmilylmk <liumingkai@huawei.com>
---
 BUILD.gn | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/BUILD.gn b/BUILD.gn
index 9ecb37a..a6001c9 100755
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -198,12 +198,13 @@ if (defined(ohos_lite)) {
   ohos_shared_library("mbedtls_shared") {
     public_configs = [ ":mbedtls_config" ]
     output_name = "mbedtls"
-    subsystem_name = "common"
+    subsystem_name = "communication"
     part_name = "dsoftbus_standard"
     sources = mbedtls_sources
   }
 
   group("mbedtls") {
+    public_configs = [ ":mbedtls_config" ]
     deps = [ ":mbedtls_shared" ]
   }
 }
-- 
2.34.1

